tests/pytests/test.py

```diff
@@ -3822,7 +3822,7 @@ def testUsesCounter(env):
     env.cmd('ft.info', 'idx')
     env.cmd('ft.search', 'idx', '*')
 
-    assertInfoField(env, 'idx', 'number_of_uses', 3)
+    assertInfoField(env, 'idx', 'number_of_uses', 3 if not env.isCluster() else 4)
```

Comment:
because of the coordinator's usage?

Reply:
Yes, I think I'll make the cluster reference invisible to this.. Although then we can miss some references. Still seems clearer.